hackerrank/simple-array-sum/challenge.js

```diff
@@ -0,0 +1,24 @@
+// Solve the challenge in this JavaScript file
+
+const simpleArraySum = (ar) => {
+  let sum = ar;
+  sum = 1 + 2 + 3 + 4 + 10 + 11;
+  return sum;
```

Comment:
You didn't do anything with the `ar` parameter? You are supposed to sum up the element interactively